solutions.puml

```diff
@@ -52,6 +56,7 @@ hide empty description
     Solution4 --> Solution5
     Solution5 --> Solution6
     Solution6 --> Solution7
+    Solution6 --> Solution9
     Solution7 --> Solution8
     Solution8 --> [*]
   }
```

Comment:
Would you mind also rendering this for the main readme? `https://github.com/CamiloTerevinto/road-to-orleans/tree/feature/149/solutions`